### ISSUE TITLE
Error/Bug - Suspicious activity detected from your network. Please try again soon

### ISSUE THREAD

**Issue by Corwin17** [Windscribe/browser-extension#14]
I installed the extension and made an account and everything worked great! I loved the extension and all but upon trying to sign back in after accidentally signing out, I get an error message saying "Suspicious activity detected from your network. Please try again soon". I've been trying to log in for a couple weeks now with the same error regardless of from what computer I try to log in from etc. etc.
</br>The error in question:
![image](https://github.com/Windscribe/browser-extension/assets/117132386/8083d621-ac4e-4528-9a74-3ce658510bc9)

**Comment by 1RandomDev** [comment]
Did you find any solution? I constantly get the same error when trying to request the 'ServerConfigs' endpoint in NodeJS via the ws-api-client.

**Comment by Corwin17** [comment]
I found a really weird solution that worked consistently for me but I'm not sure if it'll work for you. Basically what I did is turned off my wifi, tried to sign in again so it'd say there was no connection, and then turned my wifi back on while rapidly clicking the sign in button and for some reason it let me sign in then. I don't know why that works but I found that anytime I encountered that issue on any of my devices while trying to sign in, I could just do that and it worked.
